### DISCUSSION TITLE
FoxESS - Grid Consumption & FeedIn - Unknown - Other sensors working

### DISCUSSION THREAD

**Discussion by coalfield** [macxq/foxess-ha#158] [Q&A]
Have this up and running although my grid consumption is showing Unknown, I know the integration is working because other sensors are working but this is the sensor required for the energy dashboard.  Anyone have any ideas?  I simply followed a fresh install...

Error showing on the Enegry Dashboard config:

statistics_not_defined
sensor.foxess_grid_consumption

**Comment by 0superolo0** [comment]
Got the same error

**Comment by pa3cello** [comment]
Same error, HA docker install 2023.10.2
